popup.js

```diff
@@ -0,0 +1,166 @@
+/* eslint linebreak-style: ["error", "windows"] */
+/* eslint linebreak-style: ["error", "unix"] */
```

Comment:
- [ ] Kindly remove this comment. I noticed that you don't have an `eslintrc.json` file for this branch. The essence of `eslint` is to make you use Javascript best practices when writing Javascript codes. I would like to suggest that you make the `linters` file trackable by GitHub by removing the `*.json` in your .gitignore file. Those `linters` files provided by Microverse will help you spot errors in your code when installed on your local machine. You can use `npx eslint . --fix` to automatically fix all eslint errors. Then change your linebreak from _**CRLF**_ to _**LF**_ on your visual code editor.

  - Use the image below as a guide to change it on your code editor.
 
![Image](https://user-images.githubusercontent.com/77106962/152647820-ff6d53e3-b00c-4355-a60e-395e5c14ec12.png)